salsa_dancing_molecules/command_plot.py

```diff
@@ -0,0 +1,60 @@
+"""Function that decides to plot single-plot or four-plots."""
+from salsa_dancing_molecules.variables import Variables
+from salsa_dancing_molecules.scatter_plot import (draw_scatter_plot,
+                                                  draw_four_scatter_plots)
+import numpy as np
+import matplotlib.pyplot as plt
+from pandas import *
+
+
+def data_plot(Q='-', filename='simulation_data.csv', newfile='fourplots.png'):
+    """
+    Generate one or four plots depeding on inarguments.
+
+    Q : Quanitity, like {Ekin, Epot, cnve, Pres, Temp}
+
+    Filename: The filename containg the data
+
+    Newfile: The name of the file the image will be saved to
+    """
+    data = read_csv(filename)
+    Var = Variables()
+    Var.timesteps = data.columns[-1]
+    Var.potential_energies = data['Potential Energy (eV)'].tolist()
+    Var.kinetic_energies = data['Kinetic Energy (eV)'].tolist()
+    Var.atomic_pressures = data['Pressure (Pa)'].tolist()
+    Var.temperatures = data['Temperature (K)'].tolist()
+    Var.specific_heat_NVE = data[
+        'Specific Heat Capacity NVE (J/[K kg])'].tolist()
+
+    time_step = np.linspace(0, int(Var.timesteps),
+                            len(Var.potential_energies))
+
+    if (Q != '-'):

```

Comment:
Python does not require parentheses in if statements. We haven't used them elsewhere.